client/source/components/RR/ReviewTile.jsx

```diff
@@ -60,66 +52,71 @@ const ReviewTile = function ReviewTile({ review, updateList }) {
   };
 
   return (
-    <div className="review-tile">
-      <div className="review-header">
-        <span className="review-rating">{convertRating(review.rating)}</span>
-        <span className="review-user-date">
-          {`${review.reviewer_name}, `}
-          {formattedDate}
-        </span>
-      </div>
-      <div className="review-summary">{capSummary(review.summary)}</div>
-      <div className="review-body-container">
-        <div className="review-text">{capBody(review.body)}</div>
-        { review.body.length > 250 && fullBody === false
+    <ReviewTileContainer>
```

Comment:
Not a merge blocker. There is a _lot_ going on in this render section. I'd suggest breaking it up into smaller sub-components.